apidocs/conf.py

```diff
@@ -0,0 +1,51 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import os
+import sys
+
+# Add custom paths for modules
+sys.path.insert(0, os.path.abspath("../plugins"))
+sys.path.insert(1, os.path.abspath("../scripts"))
+sys.path.insert(2, os.path.abspath("../Surfactant"))
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "Surfactant Documentation"
+copyright = "2024, surfactant"
+author = "surfactant"
+release = "1.0.0"
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = [
+    "sphinx.ext.autodoc",  # Automatically generate documentation from docstrings
+    "sphinx.ext.napoleon",
+    "sphinx.ext.viewcode",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.githubpages",
```

Comment:
Instead of adding these to a new file, they can be added to the existing docs site extensions: https://github.com/LLNL/Surfactant/blob/7f2498e2dccf23bd7a5554bd923007d006daaf0b/docs/conf.py#L17

Some PyPI packages may also need to be listed in https://github.com/LLNL/Surfactant/blob/main/docs/requirements.txt to make sure the ReadTheDocs machines that build the docs site have the extensions installed.